src/any.rs

```diff
@@ -692,6 +716,14 @@ impl Dynamic {
             };
         }
 
+        #[cfg(feature = "decimal")]
+        if type_id == TypeId::of::<Decimal>() {
```

Comment:
Again, consider `unsafe_try_cast` if `Decimal` is `Copy`

Reply:
`unsafe_try_cast` here should be fine!

Reply:
`unsafe_try_cast` version is pushed.